config-editor/config-editor-ui/src/app/services/store/config-store-state.builder.ts

```diff
@@ -138,15 +138,15 @@ export class ConfigStoreStateBuilder {
         f.author.toLowerCase().startsWith(lowerCaseSearchTerm) ||
         f.tags === undefined
           ? true
-          : f.tags.join(' ').includes(lowerCaseSearchTerm)
+          : f.tags.join(' ').toLowerCase().includes(lowerCaseSearchTerm)
```

Comment:
can you extract this filter logic into separate function in order to not duplicate the code?